classes/options-general.php

```diff
@@ -500,23 +500,23 @@ class='howto'><?php echo $api_how_to; ?></span>
 																?>
 															</select><br/>
 															<small><span
-																	class='howto'><?php _e( 'Select "No Limit" to display all results within the search radius.', 'simplemap' ); ?></span>
+																	class='howto'><?php _e( 'Select "No Limit" to display all results within the search radius.', 'SimpleMap' ); ?></span>
 															</small>
 														</td>
 													</tr>
 
 													<tr valign="top">
 														<td><label
-																for="autoload"><?php _e( 'Auto-Load Database', 'simplemap' ); ?></label>
+																for="autoload"><?php _e( 'Auto-Load Database', 'SimpleMap' ); ?></label>
 														</td>
 														<td>
 															<select name="autoload" id="autoload">
 																<option
-																	value="none" <?php selected( $autoload, 'none' ); ?>><?php _e( 'No auto-load', 'simplemap' ); ?></option>
+																	value="none" <?php selected( $autoload, 'none' ); ?>><?php _e( 'No auto-load', 'SimpleMap' ); ?></option>
 																<option
-																	value="some" <?php selected( $autoload, 'some' ); ?>><?php _e( 'Auto-load search results', 'simplemap' ); ?></option>
+																	value="some" <?php selected( $autoload, 'some' ); ?>><?php _e( 'Auto-load search results', 'SimpleMap' ); ?></option>
 																<option
-																	value="all" <?php selected( $autoload, 'all' ); ?> <?php disabled( $disabled_autoload ); ?>><?php _e( 'Auto-load all locations', 'simplemap' ); ?></option>
+																	value="all" <?php selected( $autoload, 'all' ); ?> <?php disabled( $disabled_autoload ); ?>><?php _e( 'Auto-load all locations', 'SimpleMap' ); ?></option>
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 223 characters

<!-- Comment by [SideCI](https://sideci.com) -->